dev-qt/qtwebengine/qtwebengine-5.7.0.ebuild

```diff
@@ -70,6 +70,11 @@ DEPEND="${RDEPEND}
 	sys-devel/bison
 "
 
+PATCHES=(
+	"${FILESDIR}/${PN}-5.7.0-icu58.patch"
+	"${FILESDIR}/${PN}-5.7.0-unset-madv_free.patch"
```

Comment:
this patch is basically this upstream commit: http://code.qt.io/cgit/qt/qtwebengine-chromium.git/commit/?h=53-based&id=fa8cdb3a32c377b6290d0a92d2522186bcd48293

Would you mind backporting the offical commit or at least mention the commit SHA somewhere, so that it's easier to track in the future?